Models/SimpleMaterial/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": 2,
+    "legal": [
+        {
+            "license": "CC0",
+            "licenseUrl": "https://creativecommons.org/publicdomain/zero/1.0/legalcode",
+            "artist": "javagl",
+            "year": "2017",
+            "owner": "Public",
+            "what": "Everything",
+            "text": "CC0 1.0 Universal",
+            "spdx": "CC0-1.0",
+            "icon": "https://licensebuttons.net/p/zero/1.0/88x31.png"
+        }
+    ],
+    "tags": [
+        "core",
+        "testing",
+        "written"
+    ],
+    "screenshot": "screenshot/screenshot.png",
+    "name": "SimpleMaterial",
```

Comment:
I think this name field should have a space. `"Simple Material"`

Reply:
Actually this repo is very inconsistent about spaces in proper model names.  But it has some, for example:

https://github.com/KhronosGroup/glTF-Sample-Assets/blob/497c9c9f3c254bfce4df64e0fdbac49ce1d8a172/Models/BoxTexturedNonPowerOfTwo/metadata.json#L33